libs/wingsdk/src/core/app.ts

```diff
@@ -47,6 +47,11 @@ export interface AppProps {
    * @default false
    */
   readonly isTestEnvironment?: boolean;
+
+  /**
+   * App's external context
+   */
+  readonly sourceDir: string;
```

Comment:
Seems like should actually be `entrypointDir`, right? It seems like this is used by the `Website` resource to find the website assets relative to the entrypoint.

Which begs the question, now that we have support for multiple files, how would that work?

@Chriscbr any thoughts?

It feels like this should somehow be implicitly passed when `new cloud.Website` is called. Maybe we can add some magic keyword argument called `__dirname`. If the compiler sees it, it will assign the file's directory to it implicitly.

Something like:

```js
new cloud.Website();

// implies
new cloud.Website(__dirname: __dirname);
```

Reply:
Opened an issue: https://github.com/winglang/wing/issues/3736